ext/odbc/tests/odbc_field_name_001.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+odbc_field_name(): Getting the name of the field
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+
+odbc_exec($conn, 'CREATE TABLE field_name (foo INT, bar INT, baz INT)');
+odbc_exec($conn, 'INSERT INTO field_name VALUES (1, 2, 3)');
```

Comment:
Is this insert actually necessary? Seems we are querying meta data?

Reply:
No, it's not! Good catch!